css-pseudo-classes/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>CSS Pseudo Classes</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <p class="title">Hover Button</p>
+    <button id="button">I'm a button!</button>
```

Comment:
Since you only have 1 `button`, I probably wouldn't give this an `id`. If you gave this an `id` for purely styling sake, it would've been better to give an element a class instead. You want to apply styles to classes rather than `id`'s. and we'll learn about why a little later